jpsreport/scripts/txt2txt.py

```diff
@@ -1,34 +0,0 @@
-#!/usr/bin/python
```

Comment:
I would keep this script. From time to time you need this script for very old trajectory files.

Reply:
1bcc245d2241c5f2c5773e41c91fb7fbb2bf632d